logexp/cluster/job.yaml

```diff
@@ -0,0 +1,70 @@
+# Template job config for running the log exporter on a whole kubernetes cluster.
+# We ensure that exactly one logexp pod runs per node using hard inter-pod anti-affinity.
+# Steps:
+#   1. Prepare service-account.json file (that should have write access to specified GCS path)
+#   2. `kubectl create secret generic logexp-service-account --from-file=service-account.json=/path/to/service-account.json`
+#   3. Fill in this template with the environment variable values
+#   4. `kubectl create -f job.yaml`
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: logexp
+spec:
+  parallelism: {{.NumNodes}}
+  template:
+    metadata:
+      labels:
+        app: logexp
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - logexp
+            topologyKey: kubernetes.io/hostname
+      restartPolicy: OnFailure
+      containers:
+      - name: logexp-test
+        image: gcr.io/google-containers/logexp:latest
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        command:
+        - logexp
+        - --node-name=$(NODE_NAME)
+        - --kubernetes-provider={{.KubernetesProvider}}
```

Comment:
Hmm, I still don't understand where `KubernetesProvider` and `GcloudAuthFilePath` come from?